collect_app/src/androidTest/java/org/odk/collect/android/injection/AndroidTestComponent.java

```diff
@@ -0,0 +1,57 @@
+package org.odk.collect.android.injection;
+
+
+import android.app.Application;
+
+import org.odk.collect.android.GuidanceHintFormTest;
+import org.odk.collect.android.ResetAppStateTestCase;
+import org.odk.collect.android.SharedPreferencesTest;
+import org.odk.collect.android.dao.FormsDaoTest;
+import org.odk.collect.android.injection.config.AppComponent;
+import org.odk.collect.android.injection.config.AppModule;
+import org.odk.collect.android.injection.config.scopes.PerApplication;
+import org.odk.collect.android.settings.QrCodeTest;
+import org.odk.collect.android.tasks.DownloadFormListTaskTest;
+import org.odk.collect.android.tasks.InstanceServerUploaderTest;
+import org.odk.collect.android.utilities.ImageConverterTest;
+import org.odk.collect.android.utilities.WebUtilsTest;
+
+import dagger.BindsInstance;
+import dagger.Component;
+import dagger.android.support.AndroidSupportInjectionModule;
+
+@PerApplication
+@Component(modules = {
+        AndroidSupportInjectionModule.class,
+        AppModule.class,
+        AndroidTestModule.class
+})
+public interface AndroidTestComponent extends AppComponent {
```

Comment:
There's a `TestComponent` coming in from the SMS Submission branch soon. So we are gonna have to merge these two when both of these are eventually merged in. Just putting this here as a reminder.

Reply:
Yes I will rebase this branch once sms submission PR is merged